### PR TITLE
EvaluatedModel: Properly decompose the declared license

### PR DESCRIPTION
**Pull request by fviernau**
The licenses array contains entries for declared licenses and detected licenses while only the entries for the detected licenses were actually decomposed. Use `decompose()` instead of `licenses()` also for deriving the entries for the declared licenses.

### CODE REVIEW

File: reporter/src/funTest/assets/static-html-reporter-test-expected-output.html [hunk 1]
```diff
@@ -424,7 +424,7 @@
 </script>
     <div id="report-container">
       <div class="ort-report-label">Scan Report</div>
-      <div>Created by <strong>ORT</strong>, the <a href="http://oss-review-toolkit.org/">OSS Review Toolkit</a>, version <REPLACE_ORT_VERSION> on <REPLACE_TIMESTAMP>.</div>
+      <div>Created by <strong>ORT</strong>, the <a href="http://oss-review-toolkit.org/">OSS Review Toolkit</a>, version 0.1.0-SNAPSHOT on <REPLACE_TIMESTAMP>.</div>
```
Comment (mnonnenmacher) [inline reporter/src/funTest/assets/static-html-reporter-test-expected-output.html:427]: This should stay `<REPLACE_ORT_VERSION>`.

File: reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "id" : "GPL-3.0-only WITH GCC-exception-3.1 AND CC-BY-NC-3.0"
   }, {
     "_id" : 6,
-    "id" : "GPL-3.0-only"
+    "id" : "GPL-3.0-only WITH GCC-exception-3.1"
```
Comment (mnonnenmacher) [inline reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json:43]: Commit message: First sentence should end with "...handling of detected licenses", right?
Nit: "pior" -> "prior"

Comment (fviernau) [reply reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json:43]: Done.

File: reporter/src/funTest/assets/static-html-reporter-test-expected-output.html (hunk 1 above)
Comment (fviernau) [reply reporter/src/funTest/assets/static-html-reporter-test-expected-output.html]: Done.

**Comment by fviernau** [comment]
On appveyor and travis only the unrelated GoDep test failed, while the relevant reporter tests passed. So merge regardlessly.

**Pull request merged**